tool/tsh/kube.go

```diff
@@ -226,10 +225,12 @@ func (c *kubeLoginCommand) run(cf *CLIConf) error {
 	profile, _, err := client.Status("", cf.Proxy)
 	if err != nil {
 		if trace.IsNotFound(err) {
-			fmt.Println("Not logged in.")
-			return nil
+			return trace.Errorf("not logged in")
```

Comment:
Would it make sense to return `trace.AccessDenied()` here and below instead? Seems fitting.